yt/yt/server/queue_agent/queue_static_table_exporter.cpp

```diff
@@ -209,79 +363,124 @@ class TQueueExportTask
             "Error committing main export task transaction for queue %v",
             Queue_);
 
+        YT_LOG_DEBUG("Finished creating exported tables (PreparedTableCount: %v, ExportedTableCount: %v, SkippedTableCount: %v)",
+            TaskParts_.size(),
+            exportedTableCount,
+            skippedTableCount);
+
         ProfilingCounters_->ExportedRows.Increment(diff.RowCount);
         ProfilingCounters_->ExportedChunks.Increment(diff.ChunkCount);
-        ProfilingCounters_->ExportedTables.Increment();
+        ProfilingCounters_->ExportedTables.Increment(exportedTableCount);
+        ProfilingCounters_->SkippedTables.Increment(skippedTableCount);
```

Comment:
Can we add a gauge for number of "pending" exports? Say, if exports are failing for 5 export instants already, it should be equal to 5. 

~queue_size for a lot of our services/throttlers